azure-quantum/azure/quantum/optimization/solvers.py

```diff
@@ -248,25 +288,60 @@ def check_set_positive_int(self, var: int, var_name: str):
             self.set_one_param(var_name, var)
 
     def check_set_float_limit(
-        self, var: float, var_name: str, var_limit: float
+        self, var: float, var_name: str,
+        lower_bound_exclusive: Optional[float] = None,
+        lower_bound_inclusive: Optional[float] = None,
     ):
-        """Check whether the var parameter is a float larger than var_limit.
+        """Check whether the var parameter is a float satisfying bounds.
         :param var:
             var paramter to be checked
             whether is a float larger than var_limit.
         :param var_name:
             name of the variable.
-        :var_limit:
-            limit value of the variable to be checked.
+        :lower_bound_exclusive:
+            lower limit value of the variable to be checked (exclusive)
+        :lower_bound_inclusive:
+            lower limit value of the variable to be checked (inclusive)
         """
         if not (var is None):
             if not (isinstance(var, float) or isinstance(var, int)):
                 raise ValueError(f"{var_name} shall be float!")
-            if var <= var_limit:
+            self.check_limit(
+                    var, var_name, lower_bound_exclusive=lower_bound_exclusive,
+                    lower_bound_inclusive=lower_bound_inclusive)
+            self.set_one_param(var_name, var)
+
+    def check_limit(
+        self, var: Optional[float], var_name: str,
+        lower_bound_exclusive: Optional[float] = None,
+        lower_bound_inclusive: Optional[float] = None,
+    ):
+        """Check whether the var parameter is a float satisfying bounds.
+        :param var:
+            var paramter to be checked
```

Comment:
Typo here; also what is "the var parameter" (same question as above)?

Reply:
not introduced in this PR, but fixed this copy pasted section and both of the original occurrences in line 296ff and 282ff

Reply:
Thank you!